src/xcode/ENA/ENA/Source/AppDelegate & Globals/Globals.swift

```diff
@@ -19,6 +19,32 @@ var isScreenshotMode: Bool {
 
 #endif
 
+/// Determines the CWA Hibernation State, beginning on 01.06.2023.
+var isHibernationState: Bool {
+	#if RELEASE
+
+	var hibernationStartDateComponents = DateComponents()
+	hibernationStartDateComponents.year = 2023
+	hibernationStartDateComponents.month = 6
+	hibernationStartDateComponents.day = 1
+	hibernationStartDateComponents.hour = 0
+	hibernationStartDateComponents.minute = 0
+	hibernationStartDateComponents.second = 0
+	hibernationStartDateComponents.timeZone = .utcTimeZone
+	
+	guard let hibernationStartDate = Calendar.current.date(from: hibernationStartDateComponents) else {
+		fatalError("The hibernation start date couldn't be created.")
```

Comment:
Will this fatalError lead the app to crash if the hibernation start date couldn't be created?

Reply:
Yes, because that is just a theoretically `guard` `else` case. The `date(from:)` method from `Calendar` returns an `optional` value. The `DateComponents` above are correct, so that case will never be happen.

Reply:
Ok, sounds good! Thank you, Felix!